modules/virt-deleting-virt-cli.adoc

```diff
@@ -4,45 +4,39 @@
 
 :_content-type: PROCEDURE
 [id="virt-deleting-virt-cli_{context}"]
-= Deleting {VirtProductName}
+= Uninstalling {VirtProductName} by using the CLI
 
-You can delete {VirtProductName} by using the CLI.
+You uninstall {VirtProductName} by deleting the `HyperConverged` custom resource (CR), the {VirtProductName} Operator subscription, and the `ClusterServiceVersion` (CSV) of the Operator from the `openshift-cnv` namespace.
 
 .Prerequisites
 
-- Install the OpenShift CLI (`oc`).
-
-- Access to a {VirtProductName} cluster using an account with
-`cluster-admin` permissions.
-
-[NOTE]
-====
-When you delete the subscription of the {VirtProductName} operator in the OLM by using the CLI, the `ClusterServiceVersion` (CSV) object is not deleted from the cluster. To completely uninstall {VirtProductName}, you must explicitly delete the CSV.
-====
+- Access to an {product-title} cluster using an account with `cluster-admin` permissions.
+- `oc` command installed on workstation.
```

Comment:
I have seen this written as `You have installed the OpenShift CLI (oc).` in most of our existing procedures. Suggest rewording it for consistency.

Reply:
I did see this format used elsewhere in the docs. For example, in the Operators book, [Deleting Operators from a cluster using the CLI](https://docs.openshift.com/container-platform/4.11/operators/admin/olm-deleting-operators-from-cluster.html#olm-deleting-operator-from-a-cluster-using-cli_olm-deleting-operators-from-a-cluster).
I do think Shikha's suggestion is used far more frequently.

Reply:
OK. I copied it from an existing module (deleting CSV in the Operators section)